client/src/main/java/io/spine/client/Filters.java

```diff
@@ -112,12 +109,12 @@ public static Filter eq(String fieldPath, Object value) {
      * Creates a new equality {@link Filter}.
      *
      * @param column
-     *         the entity column to filter by
+     *         the column to filter by
      * @param value
      *         the requested value
      * @return a new instance of {@code Filter}
      */
-    public static Filter eq(EntityColumn column, Object value) {
+    public static Filter eq(Column<?, ?> column, Object value) {
```

Comment:
Now, as we have the `Column` type parameterised with the type of values (`V`( as the second parameter, can we bind the type of values passed to this and similar methods below?